tests/components/blebox/test_sensor.py

```diff
@@ -17,6 +18,24 @@
 from .conftest import async_setup_entity, mock_feature
 
 
+@pytest.fixture(name="airsensor")
+def airsensor_fixture():
+    """Return a default air quality fixture."""
+    feature = mock_feature(
+        "sensors",
+        blebox_uniapi.sensor.AirQuality,
+        unique_id="BleBox-airSensor-1afe34db9437-0.air",
+        full_name="airSensor-0.air",
+        device_class="pm1",
+        unit="concentration_of_mp",
+        native_value=None,
+    )
```

Comment:
Can the test setup all sensors?

Reply:
@emontnemery what do You mean? All existing sensor classes from blebox_uniapi are included. Can You elaborate more?